packages/project-account-switcher/src/presenters/ProjectAccountSwitcherPresenter.js

```diff
@@ -0,0 +1,224 @@
+    // eslint-disable-next-line react/no-shadow
+    const activeProject = this.props.projects.filter(
+      project => project.id === this.props.activeProjectId
+    )[0];
+    return `${activeAccount.label} / ${activeProject.label}`;
+  }
+
+  constructLabelPlaceholder() {
+    // eslint-disable-next-line react/no-shadow
+    const activeAccount = this.props.accounts.filter(
+      account => account.id === this.props.activeAccountId
+    )[0];
+    // eslint-disable-next-line react/no-shadow
+    const activeProject = this.props.projects.filter(
+      project => project.id === this.props.activeProjectId
+    )[0];
+    return `${ProjectAccountSwitcherPresenter.constructPlaceholder(
+      activeAccount.label
+    )}/${ProjectAccountSwitcherPresenter.constructPlaceholder(
+      activeProject.label
+    )}`;
+  }
+
+  accountsList() {
+    return this.props.accounts.map(account => (
+      <div
+        className="project-account-switcher__item
+                      project-account-switcher__item--account
+                      project-account-switcher__item--active"
+        data-account-id={account.id}
+        onClick={this.props.onAccountClick}
+        role="button"
+        tabIndex="0"
+      >
+        <span className="project-account-switcher__item__image-wrapper">
+          <span
+            className="project-account-switcher__item__image-placeholder"
+            data-account-id={account.id}
+            dangerouslySetInnerHTML={{
+              __html: ProjectAccountSwitcherPresenter.constructPlaceholder(
+                account.label
+              )
+            }}
+          />
+        </span>
+        <span
+          className="project-account-switcher__item__label"
+          data-account-id={account.id}
+          dangerouslySetInnerHTML={{ __html: account.label }}
+        />
+      </div>
+    ));
+  }
+
+  projectsList() {
+    return this.props.projects.map(project => (
+      <div
+        className="project-account-switcher__item
+                      project-account-switcher__item--project
+                      project-account-switcher__item--active"
+        data-project-id={project.id}
+        onClick={this.props.onProjectClick}
+        role="button"
+        tabIndex="0"
+      >
+        <span className="project-account-switcher__item__image-wrapper">
+          <span
+            className="project-account-switcher__item__image-placeholder"
+            data-project-id={project.id}
+            dangerouslySetInnerHTML={{
+              __html: ProjectAccountSwitcherPresenter.constructPlaceholder(
+                project.label
+              )
+            }}
+          />
+        </span>
+        <span
+          className="project-account-switcher__item__label"
+          data-project-id={project.id}
+          dangerouslySetInnerHTML={{ __html: project.label }}
+        />
+      </div>
+    ));
+  }
+
+  flyoutContent() {
+    return (
+      <div className="project-account-switcher__lists">
```

Comment:
Not a big deal but might make more sense to use `<ul>` and `<li>` if these are actually lists.

Reply:
Awesome! Addressed in https://github.com/Autodesk/hig/pull/1102/commits/cbeb63b999c8d46a93aa5cef6c1a0e1960cdf9b3